### ISSUE TITLE
Unable to load application: NameError: uninitialized constant Rollbar::Middleware

### ISSUE THREAD

**Issue by soulim** [interagent/pliny#127]
A newly generated app doesn't want to start because of `NameError` exception in `lib/routes.rb` file.
When I explicitly require `rollbar/middleware/sinatra` everything runs perfectly.

So my `lib/routes.rb` looks like this:

```ruby
require 'rollbar/middleware/sinatra'

Routes = Rack::Builder.new do
  use Rollbar::Middleware::Sinatra
  use Pliny::Middleware::RescueErrors, raise: Config.raise_errors?
  use Pliny::Middleware::CORS
  use Pliny::Middleware::RequestID
  use Pliny::Middleware::RequestStore, store: Pliny::RequestStore
  use Pliny::Middleware::Timeout, timeout: Config.timeout if Config.timeout > 0
  use Pliny::Middleware::Versioning,
      default: Config.versioning_default,
      app_name: Config.versioning_app_name if Config.versioning?
  use Rack::Deflater
  use Rack::MethodOverride
  use Rack::SSL if Config.force_ssl?

  use Pliny::Router do
    # mount all endpoints here
  end

  # root app; but will also handle some defaults like 404
  run Endpoints::Root
end
```

Would you allow a pull request with this tiny fix? Or maybe there's a better place where I should require Rollbar middleware. I just started playing with Pliny and might miss something.

**Comment by neilmiddleton** [comment]
I don't have Rollbar in my routes (or anywhere for that matter).  I simply created an initializer for Rollbar and everything seems to work fine.

`config/initializers/rollbar.rb`:
```ruby
Rollbar.configure do |config|
  config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
  config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
  config.use_sucker_punch
  config.environment = ENV["RACK_ENV"]
end
```

**Comment by soulim** [comment]
@neilmiddleton does that also mean you don't have `use Rollbar::Middleware::Sinatra` in routes?

Without `require 'rollbar/middleware/sinatra'` in my routes, puma can't preload the app and it fails with `"Unable to load application: NameError: uninitialized constant Rollbar::Middleware"`.

**Comment by nzoschke** [comment]
A change to the Gemfile can fix it too...

```
diff --git a/Gemfile b/Gemfile
index 1ac579d..5685933 100644
--- a/Gemfile
+++ b/Gemfile
@@ -10,7 +10,7 @@ gem "pry-doc"
 gem "puma", "~> 2.10"
 gem "rack-ssl"
 gem "rake"
-gem "rollbar"
+gem "rollbar", require: "rollbar/middleware/sinatra"
 gem "sequel", "~> 4.16"
 gem "sequel-paranoid"
 gem "sequel_pg", "~> 1.6", require: "sequel"
```

**Comment by soulim** [comment]
Nice idea @nzoschke!

**Comment by brandur** [comment]
Added Noah's suggested fix in #130. That file immediately requires the main `rollbar`, so Neil's call to Rollbar's configuration block in the initializer should stay safe as well.

Thanks guys!

**Comment by soulim** [comment]
Thank you @brandur!
